frontend/src/container/NewWidget/index.tsx

```diff
@@ -402,19 +402,20 @@ function NewWidget({ selectedGraph }: NewWidgetProps): JSX.Element {
 
 	return (
 		<Container>
-			<Flex justify="space-between" align="center">
-				<FacingIssueBtn
-					attributes={{
-						uuid: selectedDashboard?.uuid,
-						title: selectedDashboard?.data.title,
-						panelType: graphType,
-						widgetId: query.get('widgetId'),
-						queryType: currentQuery.queryType,
-						screen: 'Dashboard list page',
-					}}
-					eventName="Dashboard: Facing Issues in dashboard"
-					buttonText="Facing Issues in dashboard"
-					message={`Hi Team,
+			<Row>
```

Comment:
IMO can we use `divs`with custom stylings ? Using `antd` components for stylings makes it hard to debug and review. Are we getting any out of the box benefit here by using `antd` components ?

Reply:
no this can be done with div with classname, made the required changes